Makefile

```diff
@@ -1,8 +1,12 @@
-.PHONY: default all requirements configure ironic ocp_run install_config clean ocp_cleanup ironic_cleanup host_cleanup cache_cleanup registry_cleanup workingdir_cleanup podman_cleanup bell
-default: requirements configure build_installer ironic install_config ocp_run bell
+.PHONY: default all requirements configure ironic ocp_run install_config clean ocp_cleanup ironic_cleanup host_cleanup cache_cleanup registry_cleanup workingdir_cleanup podman_cleanup bell assisted assisted_deployment_cleanup assisted_deployment_destroy_nodes assisted_deployment_deploy_nodes
+
+default: requirements configure build_installer ironic ocp_run bell
```

Comment:
Why is install_config removed from the default line?

Reply:
Looks like maybe just a problem with rebasing, https://github.com/openshift-metal3/dev-scripts/pull/1004 moved the install_config to a different step. That should be all you need to fix

Reply:
yap, looks like missed it. was a difficult rebase

Reply:
thanks a lot